src/validation.cpp

```diff
@@ -1306,6 +1334,15 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
         m_viewmempool.PackageAddTransaction(ws.m_ptx);
     }
 
+    // At this point we have all in-mempool ancestors, and we know every transaction's vsize.
```

Comment:
can assert that `package_with_ancestors.ancestor_counts.size() == workspaces.size()`?

Reply:
Added